### DISCUSSION TITLE
Rule linking two relays

### DISCUSSION THREAD

**Discussion by boozelclark** [arendst/Tasmota#12482] [Q&A] [answered]
Hi

I'm trying to create a rule to have two relays (2 and 4) on my Sonoff 4CH follow each other but I cant seem to get it to work.
I have tried 
`
rule1 on power2#state=ON do power4 ON on endon on power2#state=OFF do power4 OFF off endon
rule1 1
`

but it doesn't seem to work. Please can someone guide me on what i'e got wrong?

Thanks in advance

**Comment by sfromis** [comment, marked as answer]
When using #state in rules triggers, the value is numeric 0/1, not a string ON/OFF. And you have repeated on/off in the Power commands.

And if you had what you posted as one command line, the rules will not be activated, but instead have a syntax error at the end of the rule set.

Simpler rule for the same: `on power2#state do power4 %value% endon`

**Reply by boozelclark** [reply]
Thanks very much! That works perfectly
